recipes/urdfdom_headers/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "urdfdom_headers" %}
+{% set version = "1.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ros/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 2b3040a5f4d1e421b32d80540dd1d09fa0ef46c1d4152210ca8753c462b90e31
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+
+test:
+  requires:
+    - pkg-config
+  commands:
+    - pkg-config --modversion {{ name }}
+    - ls $PREFIX/include/urdf_*/*.h
```

Comment:
If there is no `.h` files I am expecting this `ls` command to exit with a non-zero exit code and the test to fail. I checked locally that this was the case with `conda-build`.

A suggestion of a better test to use for header-only packages would be more than welcome.

Reply:
You could explicitly check for a named header.

Reply:
OK I have done that in my last commit.